server/odc-service/src/main/java/com/oceanbase/odc/service/session/ConnectConsoleService.java

```diff
@@ -621,4 +634,42 @@ private Integer checkQueryLimit(Integer queryLimit) {
         return queryLimit;
     }
 
+    private OdcStatementCallBack generateEditPLSqlODCStatementCallBackForOBMysql(@NotEmpty List<SqlTuple> sqlTuples,
```

Comment:
pl editor is a special sceanrio, may extrac to seprate class to handle,
make it easy for integration-test, please add integraiton test cases